packages/account/src/Components/poi/status/upload-complete/upload-complete.tsx

```diff
@@ -18,12 +16,10 @@ export const UploadComplete = ({
         ? localize('We’ll review your documents and notify you of its status within 1 - 3 working days.')
         : localize('We’ll review your documents and notify you of its status within 5 minutes.');
 
-    const is_redirected_from_platform = isNavigationFromP2P() || isNavigationFromDerivGO();
```

Comment:
@matin-deriv This is only to check whether it's coming from those platforms and we will hide the <ContinueTradingButton />. Since we are removing <ContinueTradingButton />, the check is not needed here now 🙏 
![Screenshot 2024-10-23 at 4 41 44 PM](https://github.com/user-attachments/assets/c3e65afe-63ab-4161-90e5-921495c6454c)